src/resolver/nodeModuleLookup.ts

```diff
@@ -123,7 +134,7 @@ export function nodeModuleLookup(props: IResolverProps, parsed: IModuleParsed):
   if (!fileExists(packageJSONFile)) {
     return { error: `Failed to find package.json in ${folder} when resolving module ${parsed.name}` };
   }
-  const json = require(packageJSONFile);
```

Comment:
I am afraid that just passes and absolute path which won't affect the require logic

Reply:
Ah ok, then it probably isn't necessary. I'll try without it!